### ISSUE TITLE
Not working after updating to 1.6.0

### ISSUE THREAD

**Issue by Moelgaard85** [zignd/HTML-CSS-Class-Completion#58]
The plugin stopped working after updating to 1.6.0. Hovering over divs or classes just keeps saying 'loading', and the extention bar in the buttom either says 'CSS classes chached' or 'Looking for CSS classes on the workspace..' with no effect if clicking on them.
I have tried reloading, deactivating, reinstalling - without any effect.

**Comment by doggy8088** [comment]
Actually, this update broken all the IntelliSense features in HTML snippets.

**Comment by zignd** [comment]
Sorry guys, for some reason it's working on my Linux and Windows machines. One possible reason for this issue might the workspace size. Could it be that you guys are working on large front-end projects? I tested the extension on a project that had Bootstrap installed via NPM and on another project with Bootstrap referenced via CDN; both had some CSS classes defined in some separated files.

I'd like to ask you guys a favor, do you think you can set up an open repository that I could clone to reproduce the issue? It would be of great help.

**Comment by Moelgaard85** [comment]
I have set up an open repository for a small project that is not private and only few styles are directly used from index.html, and same problem appears here.

URL: [https://github.com/Moelgaard85/electron-app](https://github.com/Moelgaard85/electron-app)

It seems to break other plugins as well, such as git, HTMLHint etc.

**Comment by zignd** [comment]
I managed to reproduce, thanks for the help @Moelgaard85. I'll look into a solution tonight, I might have to create a [language server](https://code.visualstudio.com/docs/extensions/example-language-server).

**Comment by BorntraegerMarc** [comment]
I had to uninstall it, because the update caused so many performance problems!

**Comment by matheo** [comment]
This plugin caused frozen IDE for me after update to VSCode 1.13

**Comment by DuncanFaulkner** [comment]
I have had the same issue, though I only noticed the issue when I upgraded to 1.13.1 of vs code, once I had disabled this extension as per @otijhuis comments on Microsoft/vscode#28867 vs code stopped crashing and started working again.

ubuntu 16.04

**Comment by doncullen** [comment]
Experienced issues with it hanging also after installing extension.  Could not disable or uninstall extension due to VSC freezing every time it opened.  So after Ubuntu closed it down when it hung again, I opened terminal and did this:

`code --disable-extensions`

Then opened VSC again.  It finally was operational.  I then uninstalled the extension, then reloaded VSC.  Working fine with all other extensions enabled.
